app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanelList.vue

```diff
@@ -1,70 +1,12 @@
 <template>
   <div class="flex-col py-2 px-2.5 overflow-auto h-full flex">
-    <woot-button
+    <notification-panel-item
       v-for="notificationItem in notifications"
       v-show="!isLoading"
       :key="notificationItem.id"
-      size="expanded"
-      color-scheme="secondary"
-      variant="link"
-      @click="() => onClickNotification(notificationItem)"
-    >
-      <div
-        class="flex-row items-center p-2.5 leading-[1.4] border-b border-solid border-slate-50 dark:border-slate-700 flex w-full hover:bg-slate-75 dark:hover:bg-slate-900 hover:rounded-md"
-      >
-        <div
-          v-if="!notificationItem.read_at"
-          class="w-2 h-2 rounded-full bg-woot-500"
-        />
-        <div v-else class="w-2 flex" />
-        <div
-          class="flex-col ml-2.5 overflow-hidden w-full flex justify-between"
-        >
-          <div class="flex justify-between">
-            <div class="items-center flex">
-              <span class="font-bold text-slate-800 dark:text-slate-100">
-                {{
-                  `#${
-                    notificationItem.primary_actor
-                      ? notificationItem.primary_actor.id
-                      : $t(`NOTIFICATIONS_PAGE.DELETE_TITLE`)
-                  }`
-                }}
-              </span>
-              <span
-                class="text-xxs p-0.5 px-1 my-0 mx-2 bg-slate-50 dark:bg-slate-700 rounded-md"
-              >
-                {{
-                  $t(
-                    `NOTIFICATIONS_PAGE.TYPE_LABEL.${notificationItem.notification_type}`
-                  )
-                }}
-              </span>
-            </div>
-            <div>
-              <thumbnail
-                v-if="hasAssignee(notificationItem)"
-                :src="notificationItem.primary_actor.meta.assignee.thumbnail"
-                size="16px"
-                :username="notificationItem.primary_actor.meta.assignee.name"
```

Comment:
**Issue description**

The primary cause of the issue lies in the thumbnail section which accesses the notification assignee details from the conversation. This is the only instance where the notification assignee's details are being accessed in the unread notification panel.

**Steps tried to prevent**

I created a new notification panel list item component to access it by the computed property that extracts the primary_actor from notificationItem and then accesses the assignee details, but only if they are present.